ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/SingleEntityVulnerabilitiesTable.tsx

```diff
@@ -118,18 +118,16 @@ function SingleEntityVulnerabilitiesTable({
                                 <Td />
                                 <Td colSpan={5}>
                                     <ExpandableRowContent>
-                                        <p>{summary}</p>
-                                        <div
-                                            className="pf-u-p-md pf-u-mt-md"
-                                            style={{
-                                                border: '1px solid var(--pf-c-table--BorderColor)',
-                                            }}
-                                        >
-                                            <ImageComponentsTable
-                                                layers={image.metadata?.v1?.layers ?? []}
-                                                imageComponents={imageComponents}
-                                            />
-                                        </div>
+                                        <p className="pf-u-mb-md">{summary}</p>
+                                        <ComponentVulnerabilitiesTable
+                                            showImage={false}
```

Comment:
do we want to make this prop required/explicit? it looks like it defaults to `false` when empty in the component props above

Reply:
Ah, yeah I meant to remove this so that omitting the prop defaulted to false, and otherwise we write the shorthand `showImage` boolean value. Making it required+explicit is probably more readable though.